extensions/blocks/map/map.php

```diff
@@ -25,7 +25,83 @@
 function jetpack_map_block_load_assets( $attr, $content ) {
 	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
 
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		global $wp, $map_block_counter;
+		if ( ! $map_block_counter ) {
+			$map_block_counter = 0;
```

Comment:
```suggestion
			if ( ! isset( $map_block_counter ) ) {
				$map_block_counter = array();
			}
			if ( ! isset( $map_block_counter[ get_the_ID() ] ) ) {
				$map_block_counter[ get_the_ID() ] = 0;
			}
```

Reply:
Switched to this type of logic in https://github.com/Automattic/jetpack/pull/13405/commits/e651d6fa02b91c3734ddbef8c4dfa55e5cd1d4b1